### PR TITLE
chore: Slovenian translation update

### PR DESCRIPTION
**Pull request by dkrat7**
### SUMMARY
This PR updates the Slovenian translation files. Missing translations were added and improved some existing ones.

### TESTING INSTRUCTIONS
Change language, review translations.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by junlincc** [comment]
@dkrat7 Thank you so much for the contribution! We will ask around in the community to get some native speakers to review your PR soon. :) Do you have specific timeline when you want this to be merged?

**Comment by dkrat7** [comment]
Yes, it would be nice to find someone to review the translations (I think there are not many Slovenian native speakers around here). There is no specific timeline. Translations are always few steps behind the main development, it is hard to follow (I am glad that's the case:).

**Comment by villebro** [comment]
@dkrat7 this is great work, thanks so much! I'm happy to merge this as-is unless there are reviews from native Slovenian speakers forthcoming (we can always do a follow-up if this needs fine tuning)

**Review by villebro — APPROVED** [review]
